plugins/search/providers/elastic_search/lib/model/search/kEntrySearch.php

```diff
@@ -107,6 +107,17 @@ public function getElasticTypeName()
     public function fetchCoreObjectsByIds($ids)
     {
         $entries = entryPeer::retrieveByPKsNoFilter($ids);
+
+        // remove deleted entries from results (in case there is major lag - elastic may return IDs that are deleted in DB)
+		foreach ($entries as $key => $entry)
```

Comment:
Fix indentation.
Verify that if you have related objects such as caption on entry , and it was removed - that the result will not break.

Reply:
Due to complexity and chance of breaking flow - we decided to write a script to fix it.